src/cow-react/modules/application/containers/App/styled.ts

```diff
@@ -40,7 +39,7 @@ export const BodyWrapper = styled.div<{ location: { pathname: string } }>`
   align-items: flex-start;
 `}
 
-  ${({ theme, location }) => theme.mediaWidth.upToMedium`
-  padding: ${[Routes.SWAP].includes(location.pathname as Routes) ? '0 0 16px' : '0 16px 16px'};
+  ${({ theme }) => theme.mediaWidth.upToMedium`
+  padding:  '0 0 16px';
```

Comment:
Nitpick: extra space

```suggestion
  padding: '0 0 16px';
```

Reply:
Fixed